content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -319,20 +319,22 @@ validation error is thrown for any container sharing a name with another.
 
 ### Resource sharing within containers
 
-Given the order of execution for init, sidecar and app containers, the following rules
+Given the order of execution for regular init containers, sidecar and app containers, the following rules
 for resource usage apply:
 
-* The highest of any particular resource request or limit defined on all init
+* The highest of any particular resource request or limit defined on all regular init
   containers is the *effective init request/limit*. If any resource has no
   resource limit specified this is considered as the highest limit.
-* The Pod's *effective request/limit* for a resource is the higher of:
-  * the sum of all app containers request/limit for a resource
+* The Pod's *effective request/limit* for a resource is the sum of
+[pod overhead](/docs/concepts/scheduling-eviction/pod-overhead/) and the higher of:
```

Comment:
This link can be useful to add. But to me at least, the other changes included make this documentation less clear.